python-client/giskard/scanner/llm/unethical_bias_detector.py

```diff
@@ -0,0 +1,126 @@
+import copy
+from dataclasses import dataclass
+from typing import List, Sequence
+
+import evaluate
+from langchain import PromptTemplate
+import pandas as pd
+
+from giskard.datasets import Dataset
+
+from ...models.langchain import LangchainModel
+from ..decorators import detector
+from ..issues import Issue
+from ..logger import logger
+
+
+@detector("llm_unethical_bias", tags=["text_generation", "unethical"])
+class UnethicalBiasDetector:
```

Comment:
Uhm, we should distinguish this from the ‘ethical bias‘ that we use [elsewhere](https://docs.giskard.ai/en/latest/getting-started/key_vulnerabilities/ethics/index.html). Or maybe we should present everything in the same tab?

Anyhow, this is more or less harmful content generation more than ethical bias.